cluster/engine_test.go

```diff
@@ -460,3 +472,23 @@ func TestDisconnect(t *testing.T) {
 	// Double disconnect shouldn't cause panic
 	engine.Disconnect()
 }
+
+func TestRemoveImage(t *testing.T) {
+	engine := NewEngine("test", 0, engOpts)
+
+	imageName := "test-image"
+	dIs := []types.ImageDelete{{Deleted: imageName}}
+
+	apiClient := engineapimock.NewMockClient()
+	apiClient.On("ImageRemove", context.TODO(), mock.Anything).Return(dIs, nil)
```

Comment:
This shouldn't be `context.TODO()`, but `mock.Anything` because we might pass a non-empty context in the future. The second argument can be of a specific type though, like suggested above.